packages/itmat-ui-react/src/components/users/userDetails.tsx

```diff
@@ -73,6 +74,34 @@ export const EditUserForm: React.FunctionComponent<{ user: (IUserWithoutToken &
     if (whoamiloading) { return <p>Loading..</p>; }
     if (whoamierror) { return <p>ERROR: please try again.</p>; }
 
+    const showTimeFunc = {
+        showDate: function(timestamps: number) {
+            return new Date(timestamps).toISOString().substring(0, 10);
+        },
+        showTime: function(timestamps: number) {
+            return new Date(timestamps).toISOString().substring(11, 19);
+        }
+    }
```

Comment:
Not entirely sure but - i think from a optimisation perspective these pure functions should declared outside the function scope. or else every time this component re-renders these functions need to be recompiled and assigned. 

Not sure if the JIT compiler is smart enough to cache the result? i might be wrong